src/inverseRobot.js

```diff
@@ -7,7 +7,17 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const result = {};
+
+  for (const key in robot) {
+    if (robot[key] in result) {
+      return null;
+    } else {
+      result[robot[key]] = key;
+    }
```

Comment:
You don't need to use else if you return something because a return will immediately exit out of a function.

Reply:
understood, thanks